hoomd/md/TwoStepConstantPressureGPU.cu

```diff
@@ -194,28 +195,33 @@ gpu_npt_mtk_wrap_kernel(const unsigned int nwork, Scalar4* d_pos, int3* d_image,
         // fetch particle position
         Scalar4 postype = d_pos[idx];
         Scalar3 pos = make_scalar3(postype.x, postype.y, postype.z);
+        Scalar4 veltype = d_vel[idx];
```

Comment:
These Scalar4's are actually velocity + mass. Please rename anywhere you have `veltype` for readability.
```suggestion
        Scalar4 velmass = d_vel[idx];
```